racktables/racktables.py

```diff
@@ -14,20 +14,21 @@ def __init__(self, url, username, password):
         self.url = url
 
     def search(self, search_payload={}):
-        params="&".join("%s=%s" % (k, v) for k, v in search_payload.items())
-        req = self.s.get(
-            join(self.url, "index.php"),
-            params=params
-        )
+        params = "&".join("%s=%s" % (k, v) for k, v in search_payload.items())
```

Comment:
Why do you need this with `requests`? I am pretty sure passing the `search_payload` in the params should work out

Reply:
Please only comment on the changes made in this PR which is only adjusting the style with `ruff`. I didn't even read the changed code in detail.